crates/bevy_ui/src/render/mod.rs

```diff
@@ -585,7 +600,7 @@ pub fn extract_uinode_borders(
         let transform = global_transform.compute_matrix();
 
         extracted_uinodes.uinodes.insert(
-            commands.spawn_empty().id(),
+            commands.spawn(RenderFlyEntity).id(),
```

Comment:
Sorry for the noob question; what are all these `RenderFlyEntities`? They are just temporarily used in the render world for one frame?

Reply:
yeah ,we are currently using many temporary entities for rendering which need to be despawned after frame end.

Reply:
Imo RenderFlyEntity  -> TemporaryRenderEntity

Reply:
`TransientEntity` ?

Reply:
I think it should mention render in the name. Could also go the other way "MarkedForRenderWorldCleanup" or such.

Reply:
I still want to rename this